<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FwdSetupWidgetClass</class>
 <widget class="QWidget" name="FwdSetupWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DummySetupWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Files">
      <attribute name="title">
       <string>Files</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Sol">
         <property name="title">
          <string>Forward Solution</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd">
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_SolNameDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_SolName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Meas">
         <property name="title">
          <string>Measurement File (with compensator inforamtion)</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_8">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_MeasName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_MeasNameDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Source">
         <property name="title">
          <string>Source Space</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_2">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_SourceName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_SourceNameDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Bem">
         <property name="title">
          <string>BEM Model</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_3">
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_BemNameDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_BemName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_Mri">
         <property name="title">
          <string>MRI - Head Transformation</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_4">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_MriName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_MriNameDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_MinDistOut">
         <property name="title">
          <string>Output for omitted source space points</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_5">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_MinDistName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_MinDistOutDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_EEGModel">
         <property name="title">
          <string>EEG model</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_7">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_EEGModelFile">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_qPushButton_EEGModelFileDialog">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCheckBox" name="m_check_bIncludeMeg">
         <property name="text">
          <string>Compute MEG Solution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bIncludeEEG">
         <property name="text">
          <string>Compute EEG Solution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bComputeGrad">
         <property name="text">
          <string>Compute Gradient Solution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bFixedOri">
         <property name="text">
          <string>Fixed Orientation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bAccurate">
         <property name="text">
          <string>Acurate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bCoordframe">
         <property name="text">
          <string>Compute in MRI space (else Head space)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bUseThreads">
         <property name="text">
          <string>Use threads</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bDoAll">
         <property name="text">
          <string>Calculate forward solution in all nodes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bMriHeadIdent">
         <property name="text">
          <string>Identical head- and MRI- space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bFilterSpaces">
         <property name="text">
          <string>Filter Source Spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bScaleEegPos">
         <property name="text">
          <string>Scale Electrode Positions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_check_bUseEquivEeg">
         <property name="text">
          <string>Use equivalent source approach for EEG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="distance">
         <property name="title">
          <string>Minimum distance skull &lt;-&gt; source [mm]</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QDoubleSpinBox" name="m_doubleSpinBox_dMinDist">
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="EegScalpRadius">
         <property name="title">
          <string>EEG scalp radius of sphere model [mm]</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QDoubleSpinBox" name="m_doubleSpinBox_dEegSphereRad">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="EEGModelOrigin">
         <property name="title">
          <string>Sphere Model Origin (x,y,z) [mm]</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="m_doubleSpinBox_dVecR0x">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="m_doubleSpinBox_dVecR0z">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="m_doubleSpinBox_dVecR0y">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_qGroupBox_EEGModelName">
         <property name="title">
          <string>EEG model name</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="m_qGridLayout_Fwd_6">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="m_qLineEdit_EEGModelName">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
